build/build.bat

```diff
@@ -9,21 +9,22 @@ setlocal
 
 rem error if BOOST_ROOT not set
 set BOOST=%BOOST_ROOT%
+set PROJECT_DIR=%cd%
 
-pushd ..\test
-%BOOST%\b2.exe -q %BUILD_ARGS% %*
-popd
+cd %BOOST%
+b2.exe %PROJECT_DIR%\test -q %BUILD_ARGS% %*
 if errorlevel 1 exit /b %ERRORLEVEL%
 
-set BOOSTBOOK_DIR=..\bin\turtle\boostbook
+cd %PROJECT_DIR%
+set BOOSTBOOK_DIR=%PROJECT_DIR%\bin\turtle\boostbook
 xcopy /Y /S /Q /I %BOOST%\tools\boostbook\xsl %BOOSTBOOK_DIR%\xsl
 xcopy /Y /S /Q /I %BOOST%\tools\boostbook\dtd %BOOSTBOOK_DIR%\dtd
-xcopy /Y /S /Q /I boostbook %BOOSTBOOK_DIR%
-xcopy /Y /S /Q /I %BOOST%\doc\src\boostbook.css ..\doc\html
-xcopy /Y /S /Q /I %BOOST%\doc\src\images\*.png ..\doc\html\images
-xcopy /Y /S /Q /I %BOOST%\doc\src\images\callouts\*.png ..\doc\html\images\callouts
+xcopy /Y /S /Q /I build\boostbook %BOOSTBOOK_DIR%
+xcopy /Y /S /Q /I %BOOST%\doc\src\boostbook.css doc\html
+xcopy /Y /S /Q /I %BOOST%\doc\src\images\*.png doc\html\images
+xcopy /Y /S /Q /I %BOOST%\doc\src\images\callouts\*.png doc\html\images\callouts
 if errorlevel 1 exit /b %ERRORLEVEL%
-pushd ..\doc
-%BOOST%\b2.exe -q %*
-popd
+
+cd %BOOST%
+b2.exe %PROJECT_DIR%\doc -q %*
```

Comment:
Do we even need to `cd %BOOST%` to do this? What about
```
%BOOST%\b2.exe %PROJECT_DIR%\doc -q %*
```
?

Reply:
Yes. This was the reason why the master tests failed. https://github.com/boostorg/build/issues/622

It searches for the project-config.jam in the current directory. Hence you need to change to BOOST_ROOT.

However I think we can get rid of (some of) the top-level jam scripts in turtle and only use BOOST_ROOT (for cmake) as it is no longer required to set BOOST/BOOST_ROOT for B2 by changing into it first